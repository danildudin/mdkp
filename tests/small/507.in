5 2
18 67 99 78 40 
92 31 36 55 74 331
48 49 37 38 5 77
