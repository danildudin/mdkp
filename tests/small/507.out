177
2
2 3 
